manifests/ip_nat.pp

```diff
@@ -51,4 +51,7 @@
   if $nftables::out_https {
     include nftables::rules::out::https
   }
+  if $nftables::out_all {
```

Comment:
We could simplify the generated chain, by leaving away all the other out if that one is set.

Reply:
For sure that could make sense and I configured as :+1: 

```yaml
nftables::out_ntp:   false
nftables::out_dns:   false
nftables::out_http:  false
nftables::out_https: false
nftables::out_all:   true
```
would you consider accepting to change the default to false for all of the above? This would make the code cleaner and more obvious at the expense of backwards compatibility.

Reply:
So we added these by default, as this is what we need to open +- on every system. So it's more convenient to get it running, by just including the nftables class. The result is that you have a locked down system, that is still able to talk to the outside for the things it need (time, dns + yum repos) and all the other rules you might be added by classes or we should define a more generic way to specify rules via hiera at some point.

Now I am not sure how the code would look easier if these are false by default? The hieradata would, but it also would look easier if we only include the out rules, if out_all is false.

Reply:
Meaning:

```puppet
# basic outgoing rules
if $nftables::out_all {
  include nftables::rules::out::all
} else {
  if $nftables::out_ntp {
    include nftables::rules::out::chrony
  }
  if $nftables::out_dns {
    include nftables::rules::out::dns
  }
  if $nftables::out_http {
    include nftables::rules::out::http
  }
  if $nftables::out_https {
    include nftables::rules::out::https
  }
}
```

Reply:
Sure thing , I'll write something that disables the out rules if `out_all` is enabled.

Reply:
Exactly as you suggested - seems good.